dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -97,21 +97,6 @@
 			#interrupt-cells = <3>;
 		};
 
-		adsp_host_ipc: ace_host_ipc@73000 {
```

Comment:
what are you changing here? This seems just moving things around and not related, please remove this from the commit.

Reply:
I just moved the ipc and idc nodes so the are under interrupt controler they refer to, but ok I'll restore.